src/sentry/utils/otp.py

```diff
@@ -1,7 +1,6 @@
 import time
 import hmac
 import base64
-import qrcode
```

Comment:
Is this a deep that can be removed now?

Reply:
I hope so, wanted to check with backend folks first

Reply:
Looks reasonable to remove to me

Reply:
Can it also be removed from the getsentry dependencies as well?

Reply:
It's not being used there. Here's PR: https://github.com/getsentry/getsentry/pull/5184